lib/src/main/kotlin/com/lemonappdev/konsist/core/declaration/KoFunction.kt

```diff
@@ -49,29 +51,29 @@ class KoFunction private constructor(private val ktFunction: KtFunction) :
         type?.let { KoType.getInstance(type) }
     }
 
-    fun hasOperatorModifier() = ktFunction.modifierList?.hasModifier(KtTokens.OPERATOR_KEYWORD) ?: false
+    fun hasOperatorModifier() = hasModifiers(KoModifier.OPERATOR)
 
-    fun hasInlineModifier() = ktFunction.modifierList?.hasModifier(KtTokens.INLINE_KEYWORD) ?: false
+    fun hasInlineModifier() = hasModifiers(KoModifier.INLINE)
 
-    fun hasTailrecModifier() = ktFunction.modifierList?.hasModifier(KtTokens.TAILREC_KEYWORD) ?: false
+    fun hasTailrecModifier() = hasModifiers(KoModifier.TAILREC)
 
-    fun hasInfixModifier() = ktFunction.modifierList?.hasModifier(KtTokens.INFIX_KEYWORD) ?: false
+    fun hasInfixModifier() = hasModifiers(KoModifier.INFIX)
 
-    fun hasExternalModifier() = ktFunction.modifierList?.hasModifier(KtTokens.EXTERNAL_KEYWORD) ?: false
+    fun hasExternalModifier() = hasModifiers(KoModifier.EXTERNAL)
 
-    fun hasSuspendModifier() = ktFunction.modifierList?.hasModifier(KtTokens.SUSPEND_KEYWORD) ?: false
+    fun hasSuspendModifier() = hasModifiers(KoModifier.SUSPEND)
 
-    fun hasOpenModifier() = ktFunction.modifierList?.hasModifier(KtTokens.OPEN_KEYWORD) ?: false
+    fun hasOpenModifier() = hasModifiers(KoModifier.OPEN)
 
-    fun hasOverrideModifier() = ktFunction.modifierList?.hasModifier(KtTokens.OVERRIDE_KEYWORD) ?: false
+    fun hasOverrideModifier() = hasModifiers(KoModifier.OVERRIDE)
 
-    fun hasFinalModifier() = ktFunction.modifierList?.hasModifier(KtTokens.FINAL_KEYWORD) ?: false
+    fun hasFinalModifier() = hasModifiers(KoModifier.FINAL)
 
-    fun hasAbstractModifier() = ktFunction.modifierList?.hasModifier(KtTokens.ABSTRACT_KEYWORD) ?: false
+    fun hasAbstractModifier() = hasModifiers(KoModifier.ABSTRACT)
 
-    fun hasActualModifier() = ktFunction.modifierList?.hasModifier(KtTokens.ACTUAL_KEYWORD) ?: false
+    fun hasActualModifier() = ktFunction.hasActualModifier()
```

Comment:
Let's use hasModifiers(KoModifier.X) for every modifier property because "kt methods" may have additional behavior.

It may be worth